README.md

```diff
@@ -150,6 +150,23 @@ visualization of the labels with the visualization of your predictions:
 $ ./visualize.py --sequence 00 --dataset /path/to/kitti/dataset/ --predictions /path/to/your/predictions
 ```
 
+To directly compare two sets of data, use the `compare.py` script. It will open an interactive
+opengl visualization of the pointcloud labels.
+
+```sh
+$ ./compare.py --sequence 00 --dataset_a /path/to/dataset_a/ --dataset_a /path/to/kitti/dataset_b/
+```
+
+where:
+- `sequence` is the sequence to be accessed.
+- `dataset_a` is the path to a dataset in KITTI format where the `sequences` directory is.
```

Comment:
maybe have the parameters as `--groundtruth` and `--prediction`?

but I'm also fine as leaving it like this.

Reply:
I get where you're coming from, but changing generic parameters like "dataset_a" to "groundtruth" makes comparing outputs of different models awkward, for example when comparing the outputs of SalsaNext and RangeNet.